pagespeed/automatic/Makefile

```diff
@@ -24,7 +24,7 @@
 #
 # When running this Makefile from the 'automatic' directory then it will
 # be set automatically.
-MOD_PAGESPEED_ROOT = $(shell cd ../../..; pwd)
+MOD_PAGESPEED_ROOT = $(shell cd ../..; pwd)
```

Comment:
@jeffkaufman I just noticed I left this in, but I'm not sure this is still needed?